mungegithub/submit-queue/deployment/kubernetes/configmap.yaml

```diff
@@ -44,3 +44,4 @@ data:
   check-labels.label-file: "/gitrepos/kubernetes/labels.yaml"
   issue-categorizer.triager-url: "http://issue-triager-service:5000"
   cla.cla-status-context: ""
+  alias.alias-file: "/gitrepos/kubernetes/OWNERS_ALIASES"
```

Comment:
No. But I would expect the code to gracefully handle the missing file (rather than kubernetes gracefully handling the missing environment variable :-)